youtube_dl/extractor/soundcloud.py

```diff
@@ -469,3 +472,70 @@ def _real_extract(self, url):
             'description': data.get('description'),
             'entries': entries,
         }
+
+
+class SoundcloudSearchIE(SearchInfoExtractor, SoundcloudIE):
+    IE_NAME = 'soundcloud:search'
+    IE_DESC = 'Soundcloud search'
+    _MAX_RESULTS = 200
```

Comment:
@dstftw: it's in the API docs. 200 is the maximum number of items returned in a collection. See https://developers.soundcloud.com/docs#pagination